### ISSUE TITLE
External player stopped working

### ISSUE THREAD

**Issue by zonca** [theScrabi/NewPipe#111]
I used Newpipe with VLC as player about a month ago and was working fine. Today I reinstalled it but it doesn't work anymore.
I have the "use external player" checked, VLC installed, but videos play in the embedded player.
I'm on Nexus 6, 6.0.1, Purenexus ROM.

**Comment by chschtsch** [comment]
Seems like duplicate of #108

**Comment by theScrabi** [comment]
Try to fully delete NP, and reinstall it. Sometimes the Setup file may mess up.

**Comment by zonca** [comment]
thanks @theScrabi tried that, no luck. Still same problem. Too bad, I really want to use VLC to play videos in the background.

**Comment by zonca** [comment]
Fixed this. It was not an issue of Newpipe. It was actually using an external player, the default Android media player, but UI is very similar to embedded and I didn't notice.
The problem was that didn't ask me to choose between VLC or Android Media Player. I fixed by opening Apps settings and clearing defaults for Chrome...I guess that also includes the media player. Next time I launched newpipe, I could choose VLC.
